webapp/src/components/create_issue_menu/index.tsx

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {MouseEvent} from 'react';
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
Fix these linting errors. Somewhere, there are spaces are somewhere there are not. Mattermost uses no spacing rules in destructuring, so fix these.